com.github.ransome1.sleek.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
This should be part of your code repository, it's not specific to Flathub. Same about the icon file.

Reply:
mmhh, I checked other PR for flathub and they did it the same. But I can but those files on the source repo too, no problem.

Reply:
Bump. Same about the icon file.